### DISCUSSION TITLE
Delete revoked cert ovpn

### DISCUSSION THREAD

**Discussion by 07pepa** [pivpn/pivpn#1638] [Feature requests]
i wish to delete revoked certificated (so o do not see them in `pivpn ovpn -l` ) because they will no longer be used never 

and revoking implicates that it is not deleted (like with wiregard) 

it would be nicer if you delete it outright as well

if there are any concernc another solution to list all and list  valid cert could be solution as well

**Comment by orazioedoardo** [comment]
It's not possible to delete a revoked cert because the server must know that such client is not allowed to connect.

**Reply by 07pepa** [reply]
i thought by default cert / clients that are unknown are not allowed ....  https://openvpn.net/vpn-server-resources/how-to-revoke-a-users-certificate/ and i would be happy with just not showing it (but under the hood it would show ... so it would at least make ilusion it works the same as wg

**Reply by coolapso** [reply]
revoked and unknown are two distinct things. 

Revoked:  I know you, you were once allowed but now i don't want you in my home anymore. 
Unknown: I have never seen you, you don't get in.
